.github/workflows/dbt-checks.yaml

```
@@ -0,0 +1,57 @@
+  DBT_USER: ${{ github.actor }}
+
+jobs:
+  check-files:
+    name: Compile DBT models and format SQL
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup Python version
+        uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)